source/dlangbot/database.d

```diff
@@ -0,0 +1,50 @@
+/**
+Implements a database to keep track of the Points scored
+by Authors that fix bugs. The table has the following
+columns:
+
+Github account (Primary key)
+Total Points
+*/
+module dlangbot.database;
+
+import ae.sys.database : Database;
+
+private Database _db;
+enum table_schema = q"SQL
```

Comment:
This code should be moved to a separate DB migration folder, so that DB maintenance tasks can be performed without modifying the code.

Reply:
I don't really get it. You want to move the schema code to a different folder? I think  that a better more encapsulated implementation would just have a LeaderBoardDB class that contains a static field "schema" and other useful methods. However, at this point I'm concerned with getting the application logic right. Subsequent optimizations/refactoring will deal with OOP issues.

Reply:
Generally you have one database per application, so that means that the schema is generally centralized. (Really big applications have ORMs which allow specifying it in a per-component basis, but we don't need to worry about that.)

@PetarKirov One way I can understand your comment is that you understood that schema migration needs to be performed manually. To clarify, in `ae.sys.database` you specify migration instructions by appending it to the schema array, and then the library executes it as needed depending on the SQLite `user_version` field, so there is no need to run them by hand.